### PR TITLE
Rearranged staff photos, heirarchy then seniority

### PR DESCRIPTION
**Pull request by k8hansen**
Rearranged staff photos in order of title, then in order of seniority.

Except for me and Melissa, I didn't like her dangling off the edge there.

### CODE REVIEW

File: src/components/CulturePage/OurTeam.tsx [hunk 1]
```diff
@@ -231,31 +231,21 @@ export const OurTeam: FC<OurTeamProps> = (props) => {
...
         <TeamMemberGridItem
-          name="Reed Dunkle"
-          position="Senior Software Engineer I"
-          src={require('./images/rdunkle.jpg')}
+          name="Chris Hawkins"
+          position="Senior Software Engineer III"
+          src={require('./images/chawkins.jpg')}
         />
-        <TeamMemberGridItem
+          <TeamMemberGridItem
```
Comment (mewdriller) [inline src/components/CulturePage/OurTeam.tsx:248]: ⚠️ Looks like the formatting is a _touch_ off on this line. Might just need to run `prettier` on it again.

**Review by mewdriller — APPROVED** [review]
LGTM! :bouquet: :busts_in_silhouette:

I did have one ⚠️ comment to look into, but it should be super quick.

File: src/components/CulturePage/OurTeam.tsx (hunk 1 above)
Comment (k8hansen) [reply src/components/CulturePage/OurTeam.tsx:248]: Nice catch, I thought I got them all thanks!

**Review by JaviAir — APPROVED** [review]
LGTM!

**Pull request merged**